Mathlib/Data/Matroid/Basic.lean

```diff
@@ -329,7 +329,7 @@ section aesop
   It uses a `[Matroid]` ruleset, and is allowed to fail. -/
 macro (name := aesop_mat) "aesop_mat" c:Aesop.tactic_clause* : tactic =>
 `(tactic|
-  aesop $c* (config := { terminal := true })
+  aesop $c* (config := {terminal := true})
```

Comment:
I guess the new syntax isn't possible here?

Reply:
Yup!